Gaspar-R/precurso/bingo/script/main.js

```diff
@@ -0,0 +1,254 @@
+"use strict";
+
+//1. Al ejecutar se solicitara el nombre del usuario/jugador
+//2. Debe mostrar un carton con 5/15 numeros excluyendo el cero.
+//3. Para avanzar al siguiente turno el usuario debera confirmar mediante confirm(), visualizandose otro numero. Si coincide con alguno del carton este cambiara a X o 0.
+//4. El carton debe mostrarse al final de cada turno, indicandose si se ha encontrado algun numero y solicitando la confirmacion de un nuevo turno.
+//5. Al completar todos los numeros de una misma linea mostrara un mensaje 'LINEA'. Este evento solo ocurrira una vez.
+//6. El juego finalizara al coincidir todos los numeros.
+//7. Debera indicar en cuantos turnos se ha completado el carton.
+//8. Debera preguntarse si se desea iniciar un nuevo juego.
+
+//Variables globales
+
+let card = [[], [], []];
+let nextBall = false;
+let matchedNumbersQty;
+let matchedNumbersQtyByLine;
+let line;
+let numbersAvailableToPlay;
+let numbersAvailableToPlayIterator;
+let nextNumberToPlay;
+let turnsPlayed;
+let username;
+
+//------variables para uso en HTML---------------//
+
+let startGameBtn = document.querySelector("#startBtn");
```

Comment:
Variables's name for DOM selectors better with CONST. Change "let" variable declaration to "const".